types/swiper/index.d.ts

```diff
@@ -439,6 +439,7 @@ export interface SwiperOptions {
     wrapperClass?: string;
 
     // Components
+    controller?: ControllerOptions;
```

Comment:
@AlexReff question, the docs linked states this could be also a `boolean`:
https://swiperjs.com/api/#controller
```ts
const swiper = new Swiper(containerSelector, {
        controller: true,
```
so:
```ts
controller?: ControllerOptions | true;
```

Reply:
Good catch, I didn't even realize that when adding this.

`controller?: ControllerOptions | boolean;`

Would be correct, but I'm not sure how to update my PR.

Reply:
> Would be correct, but I'm not sure how to update my PR.

Using desktop app:
https://help.github.com/en/desktop/contributing-to-projects
simply: add new changes as a new commit and push changes (all from your branch), your commits will be squashed later into single one if accepted

If you have used the Github online editor:
https://www.youtube.com/watch?v=e41HPOHX9aE

Reply:
Thanks, updated!